### ISSUE TITLE
how i can do that? (stock in crudbooster demo)

### ISSUE THREAD

**Issue by fabiopesaju** [crocodic-studio/crudbooster#607]
This is in: Product Data > Add New Product Data
or here: https://crudbooster.com/demoo/public/admin/products/add

Which item in the documentation explains how to do this?

![screenshot](https://user-images.githubusercontent.com/20627847/27076424-8a137e98-5003-11e7-9eec-30ab1ec34838.png)

Thank you guys

**Comment by fabiopesaju** [comment]
anyone?

**Comment by caweb** [comment]
seems something like this:
https://github.com/crocodic-studio/crudbooster/blob/master/docs/en/how-to-make-a-child-form.md

**Comment by fabiopesaju** [comment]
thank you @caweb ... works a half... but do not appears the "show xx entries", and do not appears too, the search field....

**Comment by caweb** [comment]
I would like to know too if there is other example, with the features that you mentioned. Let's see if anyone else gives us information about it.

**Comment by fabiopesaju** [comment]
of course... 

@fherryfherry, can you help us?

**Comment by fherryfherry** [comment]
Unfortunately, there is no feature like in the demo for latest version

**Comment by fabiopesaju** [comment]
but... why not? when will we have?

**Comment by fherryfherry** [comment]
Yes, we have replaced it with https://github.com/crocodic-studio/crudbooster/blob/master/docs/en/how-to-make-a-child-form.md ,
